src/nbl/ext/ToneMapper/CToneMapper.cpp

```diff
@@ -13,16 +13,16 @@ using namespace nbl::video;
 using namespace ext::ToneMapper;
 
 
-core::SRange<const IGPUDescriptorSetLayout::SBinding> CToneMapper::getDefaultBindings(IVideoDriver* driver, bool usingLumaMeter)
+core::SRange<const IGPUDescriptorSetLayout::SBinding> CToneMapper::getDefaultBindings(ILogicalDevice* device, bool usingLumaMeter)
 {
-	auto lumaBindings = ext::LumaMeter::CLumaMeter::getDefaultBindings(driver);
+	auto lumaBindings = ext::LumaMeter::CLumaMeter::getDefaultBindings(device);
 	const auto inputImageBinding = lumaBindings.begin()[2];
 	if (usingLumaMeter)
 	{
 		assert(lumaBindings.size()==3ull);
 		const auto uniformBinding = lumaBindings.begin()[0];
 		const auto ssboBinding = lumaBindings.begin()[1];
```

Comment:
cannot be a static, because sampler if refcounted

Reply:
also I just told you to change the `CLumaMeter::getDefaultBindings` to non-static so that kinda settles the score

Reply:
@nipunG314 no more bindings